frontend/src/components/Step3.tsx

```diff
@@ -0,0 +1,204 @@
+                  required: true,
+                },
+                {
+                  inputTitle: "Zip",
+                  defaultMessage: "Enter Zip",
+                  inputType: "text",
+                  required: true,
+                },
+                {
+                  inputTitle: "Country",
+                  defaultMessage: "Enter Country",
+                  inputType: "text",
+                  required: true,
+                },
+                {
+                  inputTitle: "Hire Date",
+                  defaultMessage: "mm/dd/yyyy",
+                  inputType: "text",
+                  required: true,
+                },
+                {
+                  inputTitle: "Last Date Worked",
+                  defaultMessage: "mm/dd/yyyy",
+                  inputType: "text",
+                  required: true,
+                },
+              ]}
+              add={true}
+            />
+          </div>
+          <hr />
+          <div className={styles.formSection}>
+            <WorkExperienceForm
+              sectionName="Total Work Experience"
+              formInputs={[
+                {
+                  inputTitle: "Total Hours Work Experience",
+                  defaultMessage: "Number of Hours",
+                  inputType: "text",
+                  required: true,
+                },
+                {
+                  inputTitle: "Total Years Work Experience",
+                  defaultMessage: "Number of Years",
+                  inputType: "text",
+                  required: true,
+                },
+              ]}
+            />
+          </div>
+          <hr />
+          <div className={styles.formSection}>
+            <WorkExperienceForm
+              sectionName="Upload Proof of Diversified Interior Design Experience (5+ years of experience)"
+              formInputs={[]}
+            />
+
+            <p>
+              i. IF YOU ARE SELF-EMPLOYED – Complete the{" "}
+              <Link
+                to="https://ccidc.org/wp-content/uploads/2023/05/Work-Verification-Form.pdf"
+                className={styles.blueLink}
+              >
+                Employment Verification Form
+              </Link>{" "}
+              and include Tax Returns that claims yourself as an Interior Designer in the occupation
+              section going back the amount of years you are claiming experience for or letter from
+              a CPA or Attorney.
+            </p>
+
+            <p>
+              ii. IF YOU ARE NOT SELF-EMPLOYED – Fill out the{" "}
+              <Link
+                to="https://ccidc.org/wp-content/uploads/2023/05/Work-Verification-Form.pdf"
+                className={styles.blueLink}
+              >
+                Employment Verification Form
+              </Link>{" "}
+              must be signed by Candidate and Employer.
+            </p>
+
+            <Button onClick={undefined} additionalStyle={styles.upload}>
```

Comment:
Can we change the underlying button to `type=button` it is currently triggering the form submit.